skfp/fingerprints/atom_pair.py

```diff
@@ -206,14 +216,24 @@ def _calculate_fingerprint(
             countSimulation=self.count_simulation,
         )
         if self.count:
-            X = [
+            Y = [
```

Comment:
Why Y? X was good

Reply:
X is a list of molecules and I can't overwrite it with fingerprints because I need list of molecules to perform scaling

Reply:
Ah, I see. So maybe `fps` variable, as a abbreviation for `fingerprints`? I think it's quite commonly used

Reply:
That was the advantage of doing it with one list comprehension as it was before